hoomd/mpcd/SphereGeometry.h

```diff
@@ -169,11 +197,13 @@ class __attribute__((visibility("default"))) SphereGeometry
         const Scalar m_R;       //!< Sphere radius
         const Scalar m_R2;      //!< Square of sphere radius
         const boundary m_bc;    //!< Boundary condition
+	const Scalar m_V;       //!<Velocity of interface
+	const Scalar m_V2;      //!<square of interface velocity
     };
 
 } // end namespace detail
 } // end namespace mpcd
+} // end namespace detail
+} // end namespace mpcd
```

Comment:
These should not be here, how does this compile?
```suggestion
```

Reply:
My apologies for these silly mistakes